tests/conftest.py

```diff
@@ -1406,8 +1407,26 @@ def tier_marks_name():
     return tier_marks_name
 
 
+@pytest.fixture(scope="session")
+def upgrade_marks_name():
+    """
+    Gets the upgrade mark names
+
+    Returns:
+        list: list of upgrade mark names
+
+    """
+    upgrade_marks_name = []
+    for upgrade_mark in upgrade_marks:
+        try:
+            upgrade_marks_name.append(upgrade_mark().args[0].name)
+        except AttributeError:
+            log.error(f"upgrade mark does not exist")
```

Comment:
tests/conftest.py:1424:23: F541 f-string is missing placeholders

Remove f please if you are not expanding any variable here.